generators/ruby-v2/sdk/src/readme/ReadmeSnippetBuilder.ts

```diff
@@ -258,6 +261,33 @@ export class ReadmeSnippetBuilder extends AbstractReadmeSnippetBuilder {
         `);
     }
 
+    private renderCustomPaginationSnippet(endpoint: EndpointWithFilepath): string {
+        return this.writeCode(dedent`require "${this.rootPackageName}"
+
+            # For custom pagination, the response is returned directly.
+            # You can use the CustomPager utility class to help with navigation.
+            response = ${this.getMethodCall(endpoint)}(
+                ...
+            )
+
+            # Access the response data directly
+            puts "Got response: #{response}"
+
+            # If using the CustomPager utility:
+            pager = ${this.rootPackageClientName}::Internal::CustomPager.new(
```

Comment:
we should make this the actual name of the CustomPager, which comes from config. You'll need to plumb that through